plugins/github-actions/src/components/Router.tsx

```diff
@@ -31,7 +32,8 @@ export const isGithubActionsAvailable = (entity: Entity) =>
   Boolean(entity.metadata.annotations?.[GITHUB_ACTIONS_ANNOTATION]);
 
 /** @public */
-export const Router = () => {
+// Optional cardview prop
+export const Router = ({ cardView = false }: { cardView?: boolean }) => {
```

Comment:
Could we make this a string enum instead? Something like `view: 'table' | 'cards'`, and then default it to `'table'`. That allows for additional views in the future, and even expanding to view-specific options with `view: { type: 'view', ... } | ...`

Reply:
Changed to enum.